guarddog/analyzer/sourcecode/bidirectional-characters.yml

```diff
@@ -0,0 +1,20 @@
+rules:
+  - id: bidirectional-characters
+    message: This package contains contains bidirectional (bidi) characters
+    metadata:
+      description: Identify when a package contains contains bidirectional (bidi) characters. See more at https://trojansource.codes/
+    patterns:
+      - pattern-either:
+          - pattern-regex: ‪
```

Comment:
is there a better way to express this e.g. through unicode values? just worried that this rule will be hard to figure out for whoever comes next

Reply:
apparently something like `\u2069` would not work.
I can add comments above each line, WDYT?

Reply:
would be great